src/sprout/Controllers/AdminController.php

```diff
@@ -807,6 +807,189 @@ public function importAction($type)
         Url::redirect("admin/contents/{$type}");
     }
 
+    /**
+    * Shows the import form for the specified item
+    *
+    * @param string $type The type of item to show the import form of
+    **/
+    public function emailReports($type)
+    {
+        AdminAuth::checkLogin();
+
+        $view = new PhpView('sprout/admin/main_layout');
+        $this->setDefaultMainviewParams($view);
+
+        $ctlr = $this->getController($type);
+        if (! $ctlr) return;
+        if (! $this->checkAccess($ctlr, 'email_report', false)) return;
+
+        $this->setNavigation($view, $ctlr);
+
+        $main = $ctlr->_getEmailReports();
+        if ($main instanceof AdminError) {
+            $this->error($main->getMessage(), $ctlr);
+            return;
+        }
+
+        if (! is_array($main)) $main = array('title' => $ctlr->getFriendlyName(), 'content' => $main);
```

Comment:
_cough_ square-bracket-arrays _cough_

I know this is copypasta and it's harmless, so don't jump to fix it.